includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -134,11 +134,20 @@ public function init_settings() {
 	}
 
 	public function init_settings_data() {
+		// don't override/save settings on Preview requests
+		if ( isset( $_REQUEST['action'] ) && 'wpo_wcpdf_preview' === $_REQUEST['action'] ) {
+			return;
+		}
+		
 		// order
-		$this->order_settings      = $this->get_order_settings();
+		$this->order_settings  = $this->get_order_settings();
 		// pdf
-		$this->settings            = $this->get_settings();
-		$this->latest_settings     = $this->get_settings( true );
+		$this->settings        = $this->get_settings();
+		$this->latest_settings = $this->get_settings( true );
+		
+		// save settings
+		$latest = $this->use_historical_settings() ? false : true;
+		$this->save_settings( $latest );
```

Comment:
I think it would be good to directly use `$this->use_historical_settings()` instead of `$latest`.